index.html

```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="styles.css" type="text/css">
   </head>
   <body>
+    <button id="tickets">Get Tickets Here!</button>
```

Comment:
Can you change this to a link that navigates to https://www.eventbrite.com/e/hacktoberfest-in-kamloops-2018-tickets-51230704416